### ISSUE TITLE
FS errors

### ISSUE THREAD

**Issue by MReed1959** [limiter121/esp32-obd2-emulator#1]
Build and tried - first was getting brown-outs so I added a 5V PS.

Now I keep getting "W (2269) vfs_fat_spiflash: f_mount failed (13)"

It can't mount the file-system. Does it need some external memory?

Tried it on two different ESPWROOM32, ESP32D0WDQ6 (revision 1)

**Comment by MReed1959** [comment]
Found the problem. I had to make and flash the FS, and one of the utilities in the download would not run (mkfatfs).  Cloned the ESP32_mkfatfs project and re-made the utility, then copied the utility to the directory path expected.
Then, did a 'make makefatfs'
and a 'make flashfatfs'

Now I don't get the f_mount failed message, instead I get a listing of the contents of the spiflash.

At first my phone still won't connect to the web-site but then I reread the instructions (dua, it says right there to run "make flashfatfs") and it says to go to 192.168.4.1 and that worked!

THANK YOU for making this available!!!
